bff/src/Bff/Licensing/LicenseValidatorLocal.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Duende Software. All rights reserved.
```

Comment:
Consider renaming this file to match its class name (or is this intentional, because it is a partial?)